ale/drivers/uvvis_drivers.py

```diff
@@ -0,0 +1,111 @@
+import os
```

Comment:
Can this file be changed to `clementine_drivers` as this particular driver is the clementine uvvis driver?